backend/app/services/mongo/tasks.py

```diff
@@ -156,12 +104,11 @@ async def human_eval_task(
         update_payload: Dict[str, object] = {}
         update_payload["flag"] = flag
         update_payload["last_eval"] = eval_data.model_dump()
-        update_payload["human_eval.flag"] = human_eval
+        update_payload["human_eval"] = {"flag": human_eval}
```

Comment:
Instead use `= HumanEval(flag=human_eval).model_dump()`

And make HumanEval depend from ProjectElementBaseModel

Reply:
Good catch